gemfiles/activesupport3.2.gemfile

```diff
@@ -3,6 +3,7 @@
 source "https://rubygems.org"
 
 gem "activesupport", "~> 3.2.0"
+gem "memcache-client"
```

Comment:
Can you add `memcache-client` as a [development dependency](https://github.com/kickstarter/rack-attack/blob/d911a89b5ce0a28729674c34dd1bfcc791bce1b4/rack-attack.gemspec#L32-L33) instead?

This file is auto-generated from `Appraisals` file.